src/main/java/de/muenchen/isi/domain/mapper/AbfrageDomainMapper.java

```diff
@@ -175,6 +177,10 @@ void afterMappingModel2ListElementModel(
         @MappingTarget final AbfrageListElementModel abfrageListElementModel,
         final InfrastrukturabfrageModel model
     ) {
+        if (model.getAbfrage().getVerortung() != null) {
```

Comment:
Alternativ kann das Mapping im Methodenrumpf durch einen `@Mapping`-Annotation ersetzt werden.

Also entfernen des If-Statements von Zeile 180 bis 183 und Anpassen der Mapping-Annotation in Zeile 166.

` @Mapping(target = "abfrage.verortung.stadtbezirke",  target = "stadtbezirke")`